src/utils/noting/find-last-note-segment.js

```diff
@@ -1,18 +1,14 @@
-var _ = require('lodash');
 var isVFocus = require('../vfocus/is-vfocus');
 var isWithinNote = require('./is-within-note');
 var isNoteSegment = require('./is-note-segment');
 var errorHandle = require('../error-handle');
 
-module.exports = function findFirstNoteSegment(focus) {
+module.exports = function findLastNoteSegment(focus) {
 
   if (!isVFocus(focus)) {
     errorHandle('only a valid VFocus can be passed to findFirstNoteSegment, you passed: %s', focus);
   }
 
-  return _.last(
-    focus.takeWhile(isWithinNote).filter(isNoteSegment)
-  );
-
+  return focus.takeWhile(isWithinNote).filter(isNoteSegment).splice(-1)[0];
```

Comment:
Never thought of doing it that way, but that's pretty neat! To use less of lodash I guess?